src/PDFsharper.UnitTests/Pdf.IO/ParserTests.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PdfSharper.Pdf;
+using PdfSharper.Pdf.IO;
+using PdfSharper.Pdf.Filters;
+using System.Linq;
+using System.IO.Compression;
+using System.IO;
+using PdfSharper.Pdf.AcroForms;
+
+namespace PDFsharper.UnitTests.Pdf.IO
+{
+    [TestClass]
+    public class ParserTests
+    {
+
+#if DEBUG
+        [TestMethod]
+        public void Debug_FileRead()
```

Comment:
Anyone who pulls and doesn't have these documents will have an issue running these tests.